test/Toggle/GetUserTogglesForReleaseTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Volodymyr
+ * Date: 03/08/2015
+ * Time: 14:51
+ */
+
+namespace Clearbooks\Labs\Toggle;
+
+use Clearbooks\Labs\Release\Gateway\VisibleStubReleaseGateway;
+use Clearbooks\Labs\Toggle\Entity\UserToggleStub;
+use Clearbooks\Labs\Toggle\Gateway\UserToggleGatewayStub;
+
+class GetUserTogglesForReleaseTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+
+    }
+
+    /**
+     * @test
+     */
+    public function givenNoAvailableUserToggles_GetUserTogglesReturnsEmptyArray()
+    {
+        $response = $this->getUserTogglesForRelease( new UserToggleGatewayStub( [ ] ),
+            new VisibleStubReleaseGateway(), 1 );
+        $this->assertEquals( [ ], $response );
+    }
+
+    /**
+     * @test
+     */
+    public function givenInvisibleRelease_GetUserTogglesReturnsEmptyArray()
+    {
+        $unavailableToggle = new UserToggleStub( 1 );
+        $response = $this->getUserTogglesForRelease( new UserToggleGatewayStub( [ $unavailableToggle ] ),
+            new VisibleStubReleaseGateway( false ), $unavailableToggle->getRelease() );
+        $this->assertEquals( [ ], $response );
+    }
+
+    /**
+     * @test
+     */
+    public function givenAvailableUserToggles_InTheVisibleRelease_GetUserTogglesReturnsArrayOfToggles()
+    {
+        $availableToggle = new UserToggleStub( 1 );
+        $response = $this->getUserTogglesForRelease( new UserToggleGatewayStub( [ $availableToggle ] ),
```

Comment:
these lines of arrange code are quite difficult to read.

Perhaps refactor into a method which takes the array of $availableToggles in.